tests/test_rootfs.py

```diff
@@ -140,6 +141,42 @@ def test_expected_files_ext234(
                 subprocess.call(["cat", "artifact_info"])
                 raise
 
+    @pytest.mark.not_with_mender_feature("mender-convert")
```

Comment:
I would consider inverting the relevant pass conditions instead. This way, if we ever change our minds, the test framework will notify us and we can flip it it back again, plus we get assurance that the file is not added by mistake now. OTOH, if we skip the test, we might not remember to update it if we change our minds.

Reply:
That is a good idea, thanks. I will merge as-is (too many things open ATM), but I will revisit this when adding more add-ons in the near future.